web/app.php

```diff
@@ -33,7 +33,7 @@
 Request::enableHttpMethodParameterOverride();
 
 if ($_SERVER['APP_ELB']) {
-    Request::setTrustedProxies([$_SERVER['REMOTE_ADDR']], Request::HEADER_X_FORWARDED_AWS_ELB);
+    Request::setTrustedProxies([$_SERVER['REMOTE_ADDR']], Request::HEADER_X_FORWARDED_ALL);
```

Comment:
Dangerous as ELB doesn't do anything with `X-Forwarded-Host`, so could be abused. That said, we could unset it in Nginx when behind ELB.

Reply:
The current ELB seems to have no way of distinguishing between a Fastly CDN and an illegitimate request? Could allow only a whitelist in nginx?

Reply:
Sounds good. We should also rename the environment variable (`APP_CDN`?).

Reply:
Whitelist in https://github.com/elifesciences/journal-formula/pull/65